### ISSUE TITLE
Make unread shared notes similar to unread public chat

### ISSUE THREAD

**Issue by ffdixon** [bigbluebutton/bigbluebutton#7603]
To make it easier for users to see there is an update to the Shared Notes, let's place the red "unread" indicator in the same place as the public chat

![image](https://user-images.githubusercontent.com/290351/59312407-07b9e480-8c7b-11e9-90ec-e429ff16af72.png)

**Comment by pedrobmarin** [comment]
How about this approach?
![Screenshot from 2019-06-12 11-19-39](https://user-images.githubusercontent.com/1778398/59359079-1698a980-8d04-11e9-890f-0eb2595d897c.png)

The ellipsoid look of chat's unread mark looks buggy in my opinion and I believe it gets like that because of it's content. It doesn't make sense to add numbers to the notes unread marker since there is nothing obvious to quantify as changes. So, even if the marker goes to the right side of the item, it would not look like the chat one

**Comment by antobinary** [comment]
I personally like the dot approach more than the ellipsis on the right-hand side.
Let's hear from @tylercopeland

**Comment by tylercopeland** [comment]
To keep things consistent, having an indicator on the right-hand side makes the most sense. 

The part that I'm not sure about is the number in the counter. How are we defining a "new note"?  Technically a new note could be one letter, which would lead to a large number. cc @antobinary @ffdixon

**Comment by pedrobmarin** [comment]
Etherpad patches revisions to the note. These revisions are synced between all users and the html5 client checks whats the last revision the user saw before closing the note. The issue on quantifying is that a revision can add or remove content. It can be a single character, a big pasted text or even the full deletion of the note content.

I didn't find a way to implement something that could make sense for an user as an unread note notification so I chose to simply signal that something changed at the note content. The use case example I used was the user list icon that seems to do exactly the same for chat.
